core/tests/protractor_utils/ProfilePage.js

```diff
@@ -70,6 +75,9 @@ var ProfilePage = function() {
   };
 
   this.expectUserToHaveInterestPlaceholder = async function(expectedText) {
+    await waitFor.visibilityOf(
+      interestPlaceholder,
+      'Interest Place Holder is not present or Taking Time To Display');
```

Comment:
Here and throughout, please use consistent capitalization in error messages. These should be capitalized like sentences